code/modules/mob/living/carbon/brain/life.dm

```diff
@@ -3,72 +3,20 @@
 	return
 
 /mob/living/carbon/brain/handle_mutations_and_radiation()
-
-	if (radiation)
-		if (radiation > 100)
-			if(!container)//If it's not in an MMI
-				src << "<span class='danger'>You feel weak.</span>"
-			else//Fluff-wise, since the brain can't detect anything itself, the MMI handles thing like that
-				src << "<span class='danger'>STATUS: CRITICAL AMOUNTS OF RADIATION DETECTED.</span>"
-
-		switch(radiation)
-
-			if(50 to 75)
-				if(prob(5))
-					if(!container)
-						src << "<span class='danger'>You feel weak.</span>"
-					else
-						src << "<span class='danger'>STATUS: DANGEROUS LEVELS OF RADIATION DETECTED.</span>"
-		..()
+	return
 
 /mob/living/carbon/brain/handle_environment(datum/gas_mixture/environment)

```

Comment:
I prefer to err on the side of safety. So far brain mobs were completely immortal. Let's see how people react to this before adding more possible source of damage.